mne/preprocessing/ica.py

```diff
@@ -1115,6 +1115,28 @@ def _find_bads_ch(self, inst, chs, threshold=3.0, start=None,
 
         return labels, scores
 
+    def _get_ctps_threshold(self, Pk_threshold=1e-20):
+        """Automatically decide the Kuiper index threshold for CTPS method.
+
+        References
+        ----------
+        [1] Dammers, J., Schiek, M., Boers, F., Silex, C., Zvyagintsev,
+            M., Pietrzyk, U., Mathiak, K., 2008. Integration of amplitude
+            and phase statistics for complete artifact removal in independent
+            components of neuromagnetic recordings. Biomedical
+            Engineering, IEEE Transactions on 55 (10), 2353-2362.
+        """
+        N = self.info['sfreq']
+        vs = [x / 100 for x in range(1, 100)]
+        Pks = list()
+        C = np.sqrt(N) + 0.155 + 0.24 / np.sqrt(N)
```

Comment:
use sqrt from math

```
import math as m
m.sqrt
```

makes it explicit that its the sqrt of a float and not an array